### DISCUSSION TITLE
Avocado CI runs on contributor forks

### DISCUSSION THREAD

**Discussion by pevogam** [avocado-framework/avocado#5194] [RFC]
Hi all,

I recently received a series of emails from GH like:
```
Scheduled workflows are disabled automatically after 60 days of repository inactivity.

If you would like the "Setup tests" workflow to continue running, click the link bellow.

https://github.com/pevogam/avocado/actions?query=workflow%3A%22Setup+tests%22
```
Considering GH actions not wishing to run regular workflows on various forks like this and the fact that the CI has been failing for me again and again with an email about it each week, I wonder if it is not a better option for contributors to simply disable these runs.

If you are curious about the [second part](https://github.com/pevogam/avocado/runs/4510551195), I get an email per week like:
```
[pevogam/avocado] Run failed: Pre-release tests - master (08ac79f)
```
and if you note the commit, I don't apply any patches and just stay on 92.0 LTS. It could be that this commit has become outdated but I doubt the automated CI workflows are there just as a reminder that one should regularly rebase on the less stable master branch.

What are your thoughts about this?

**Comment by pevogam** [comment]
@ana Any opinion about this?

**Comment by ana** [comment]
@pevogam GitHub runs GH actions on forks by default.  IIRC the only exception are scheduled workflows. In any case, you can disable everything in your fork, see https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow

**Reply by pevogam** [reply]
> @pevogam GitHub runs GH actions on forks by default. IIRC the only exception are scheduled workflows. In any case, you can disable everything in your fork, see https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow

Yes, my concern was rather whether this is the correct thing to do. Do you agree to the statement that workflows are tied to some upstream movement and thus require regular updating in order not to have outdated and thus failing workflows?

**Reply by ana** [reply]
> > @pevogam GitHub runs GH actions on forks by default. IIRC the only exception are scheduled workflows. In any case, you can disable everything in your fork, see https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow
> 
> Yes, my concern was rather whether this is the correct thing to do. Do you agree to the statement that workflows are tied to some upstream movement and thus require regular updating in order not to have outdated and thus failing workflows?

It's your own copy of a repository, you can do what you want.
